packages/eclipse-sdk/jest.setup.js

```diff
@@ -0,0 +1 @@
+jest.setTimeout(3 * 1000 * 60)
```

Comment:
I don't know, maybe for tests stability, we have same config for solana, aptos, etc. There is not a lot of functionality for eclipse now, but maybe it will be needed in future